package.json

```diff
@@ -45,27 +46,33 @@
     "**/__tests__/fixtures/"
   ],
   "private": true,
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.{json,md,scss,yaml,yml}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
   "scripts": {
     "bootstrap": "npm-run-all -s check-versions lerna-prepare",
     "check-versions": "babel-node scripts/check-versions.js",
-    "format": "npm-run-all -p format-packages format-cache-dir format-www format-integration format-examples format-scripts format-markdown format-yaml",
-    "format-cache-dir": "prettier-eslint --write \"packages/gatsby/cache-dir/*.js\"",
-    "format-examples": "prettier-eslint --write \"examples/**/gatsby-node.js\" \"examples/**/gatsby-config.js\" \"examples/**/src/**/*.js\"",
-    "format-markdown": "prettier --write \"**/*.md\" --no-semi",
-    "format-packages": "prettier-eslint --write \"packages/*/src/**/*.js\"",
-    "format-scripts": "prettier-eslint --write \"scripts/**/*.js\"",
-    "format-www": "prettier-eslint --write \"www/*.js\" \"www/src/**/*.js\"",
-    "format-integration": "prettier-eslint --write \"integration-tests/**/src/**/*.js\" \"integration-tests/**/cypress/**/*.js\"",
-    "format-yaml": "prettier --write \"**/*.y?(a)ml\"",
+    "format": "npm-run-all -p format-packages format:js format:other",
+    "format:other": "yarn prettier --write",
+    "format:js": "eslint --ignore-path .gitignore \"**/*.{js,jsx}\" --fix",
```

Comment:
why specify the the ignore path here?

Reply:
Because otherwise it will try to parse more files as far as I know if it's not included..

Reply:
also let's put more ignored directories (in another file) instead of specifying which ones to format... there is no need to have 10 different format rules, as it's the same project.

Reply:
I'm open for better names here